lib/std/math.zig

```diff
@@ -929,6 +929,37 @@ pub inline fn fabs(value: anytype) @TypeOf(value) {
     return @fabs(value);
 }
 
+/// Fused multiply add, similar to (a * b) + c, except only rounds once,
+/// and is thus more accurate.
+/// This is the same as calling the builtin @mulAdd
+pub inline fn fma(comptime T: type, a: T, b: T, c: T) T {
+    return @mulAdd(T, a, b, c);
+}
+
+/// Returns the minimum value of a and b. This builtin accepts integers,
+/// floats, and vectors of either. In the latter case, the operation is
+/// performed element wise.
+/// This is the same as calling the builtin @minimum
```

Comment:
It's not the same as calling `@minimum` because `@minimum` accepts anytype for both parameters and the return type will depend on a combination of both types and whether each operand is comptime known.

Because of this, it kind of seems like this function should not be in std.math, in order to encourage use of the builtin.

Reply:
Perhaps these could be done in a separate pr?